package-lock.json

```diff
@@ -6523,8 +6523,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
After I did an npm install there were errors related to that command.
I then ran npm audit fix --package-lock-only to resolve those, that's why there is a change showing in this file.

Reply:
Do you know why? No other PRs seem to have required this change, so im tempted to ask to revert the file unless there is a good reason.

Reply:
I'm keeping the pr up to date with your remote master

Reply:
Hi, @craigtaub 

So I did run npm i on this project seems like the reason package lock changed was because of this?

https://github.com/npm/npm/issues/18103
```
npm i --save git+ssh://git@github.com:ajmath/npm-package-test.git and package-lock.json is created with current master sha
Push commits to ajmath/npm-package-test
Run npm i on project from step 1 and observe that package-lock.json is changed.
```

more info on this here:
https://stackoverflow.com/questions/45022048/why-does-npm-install-rewrite-package-lock-json